otelcol/collector_windows.go

```diff
@@ -141,6 +138,19 @@ func openEventLog(serviceName string) (*eventlog.Log, error) {
 	return elog, nil
 }
 
+func loggingOptionsWithEventLogCore(
```

Comment:
I extracted this to a function for 2 reasons:
1. I was trying to write a unit test for it (failed to find a way).
2. I wanted the comment explaining why the prepend is necessary to really stand out for any future contributors looking at this.